publication-commons/src/test/java/no/unit/nva/publication/model/storage/DaoTest.java

```diff
@@ -181,7 +184,8 @@ void daoCanBeRetrievedByPrimaryKeyFromDynamo(Dao originalResource) {
                 .withKey(originalResource.primaryKey()));
         Dao retrievedResource = parseAttributeValuesMap(getItemResult.getItem(), originalResource.getClass());
         
-        assertThat(originalResource, doesNotHaveEmptyValuesIgnoringFields(Set.of("data.assignee")));
+        assertThat(originalResource, doesNotHaveEmptyValuesIgnoringFields(Set.of(DATA_ASSIGNEE, DATA_FINALIZED_BY,
```

Comment:
I would try to provide tickets including these field to keep the purpose of the test instead of ignoring the new fields

Reply:
When testing we use methods that are called in production code, for example like this one:

    public static DoiRequest newDoiRequestForResource(Resource resource, Instant now) {

        var doiRequest = extractDataFromResource(resource);
        doiRequest.setIdentifier(SortableIdentifier.next());
        doiRequest.setStatus(TicketStatus.PENDING);
        doiRequest.setModifiedDate(now);
        doiRequest.setCreatedDate(now);
        doiRequest.setViewedBy(ViewedBy.addAll(doiRequest.getOwner()));
        doiRequest.setAssignee(null);
        return doiRequest;
    }


I can not add finalized values here, cause those values are generated when creating ticket.

Reply:
I think it's okay to have them in the ignoredFields list. Otherwise we are saying: finalized should never be empty, which doesn't makes sense for a ticket with status PENDING.

Reply:
The test is testing if a dao can be retrieved by a primary key from dynamo, so i don't think we are saying that finalized should never be empty if we test that a dao with finalized can be retrieved from dynamo.

I see your point with the test setup @truhacevkir, you could change the what the instanceProvider provides for the tickets, but maybe not worth the effort.